### PR TITLE
Error screens messages

### PR DESCRIPTION
**Pull request by shareef-dweikat**
---

## Code Review Checklist

- [ ] Description accurately reflects what changes are being made.
- [ ] Description explains why the changes are being made (or references an issue containing one).
- [ ] The PR appropriately sized.
- [ ] New code has enough tests. 
- [ ] New code has enough documentation to answer "how do I use it?" and "what does it do?".
- [ ] Existing documentation is up-to-date, if impacted.

### CODE REVIEW

File: lib/views/shared/receive.dart [hunk 1]
```diff
@@ -134,12 +134,27 @@ class ReceiveSharedState extends ChangeNotifier {
         switch (error.errorCode) {
           case ErrCodeTransferRejected:
             this.currentState = ReceiveScreenStates.TransferRejected;
+            this.error = "Description description ErrCodeTransferRejected";
```
Comment (donpui) [inline lib/views/shared/receive.dart:137]: @shareef-dweikat can we push all text to main/lib/constants/app_constants.dart file to have one place for edit?

File: android/app/src/main/kotlin/com/leastauthority/dart_wormhole_gui/MainActivity.kt [hunk 2]
```diff
@@ -28,6 +31,31 @@ class MainActivity : FlutterActivity() {
     private val READ_FAILURE = "5"
     private val CLOSE_FAILURE = "6"
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        when {
+            intent?.action == Intent.ACTION_SEND -> {
```
Comment (donpui) [inline android/app/src/main/kotlin/com/leastauthority/dart_wormhole_gui/MainActivity.kt:36]: This is probably related with another PR for enabling sharing with app.

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -38,6 +38,21 @@
                 <action android:name="android.intent.action.MAIN"/>
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="image/*" />
```
Comment (donpui) [inline android/app/src/main/AndroidManifest.xml:44]: We should share any file, not image only.

Comment (shareef-dweikat) [reply android/app/src/main/AndroidManifest.xml:44]: @donpui Sorry i pushed AndroidManifest.xml by accident MainActivity.kt

File: lib/views/shared/receive.dart (hunk 1 above)
Comment (shareef-dweikat) [reply lib/views/shared/receive.dart:137]: @donpui Sorry i pushed AndroidManifest.xml by accident MainActivity.kt

File: android/app/src/main/kotlin/com/leastauthority/dart_wormhole_gui/MainActivity.kt (hunk 2 above)
Comment (shareef-dweikat) [reply android/app/src/main/kotlin/com/leastauthority/dart_wormhole_gui/MainActivity.kt:36]: @donpui Sorry i pushed AndroidManifest.xml and MainActivity.kt by accident
It is temp branch, I did not know we are going to review it.

File: lib/views/shared/receive.dart (hunk 1 above)
Comment (shareef-dweikat) [reply lib/views/shared/receive.dart:137]: Yeah sure. This branch is only for app testings. If everything is working well, will invest more time to clean the code.

File: android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (donpui) [reply android/app/src/main/AndroidManifest.xml:44]: All good, free to ignore. I was testing only error messages

**Pull request closed without merging**